articles/seaborn-tutorial/index.md

```diff
@@ -0,0 +1,313 @@
+
+# How to use Seaborn for Data Visualization
+
+Data Visualization is the art of representing data in the form of graphs. It is a useful tool for professionals who work with data, i.e., financial analysts, business analysts, data analysts, data scientists, etc.
+
+In this tutorial, we will be working with Seaborn, a Python Library.
+
+  
+
+### Table of Contents
+
+  
+
+1. Introduction
+
+2. Prerequisites
+
+3. Installing Seaborn
+
+4. Import Seaborn and Load Dataset
+
+5. Different Types of Graphs
+
+6. Visualizing the Pokemon Dataset
+
+7. Conclusion
+
+  
+
+---
+
+### Introduction
+
+Seaborn is an open-source Python library build on top of matplotlib. It is used for data visualization and exploratory data analysis. Seaborn works easily with dataframes and the Pandas library. The graphs created can also be customized easily. Below are a few benefits of Data Visualization
+
+  
+
+Graphs can help us find data trends that are useful in any machine learning or forecasting project.
+
+- Graphs make it easier to explain your data to non-technical people
+
+- Visually attractive graphs can make presentations and reports much more appealing to the reader
+
+  
+
+The tutorial can be divided into three main parts. The first part will talk about installing seaborn and loading our dataset. In the second part, we will discuss some common graphs in Seaborn. Finally, we will use seaborn and the [Pokemon Data](https://www.kaggle.com/rounakbanik/pokemon) to create some cool graphs.
+
+  
+
+I will be writing all my code in a google colab file. You can find it over [here](https://colab.research.google.com/drive/1Kv8B74wBfNhCD232DTnwdOCOJdXn_IxB?usp=sharing)
+
+  
+
+---
+
+### Prerequisites
+
+1. A good understanding of Python
+
+2. Some Experience working with the Pandas Library
+
+3. Some Experience working with the Matplotlib Library, you can check out the [following article](https://www.section.io/engineering-education/getting-started-with-data-visualization-using-pandas/)
```

Comment:
A more introductory level article would be this [article](https://www.section.io/engineering-education/matplotlib-visualization-python/).